eng/CodeAnalysis.src.globalconfig

```diff
@@ -441,6 +441,9 @@ dotnet_diagnostic.CA1857.severity = warning
 # CA1858: Use 'StartsWith' instead of 'IndexOf'
 dotnet_diagnostic.CA1858.severity = warning
 
+# CA1859: Use concrete types when possible for improved performance
```

Comment:
We should also add a corresponding None entry to the .tests.globalconfig.

Reply:
Done